README.md

```diff
@@ -61,14 +37,27 @@ terraform {
   required_providers { iterative = { source = "iterative/iterative" } }
 }
 provider "iterative" {}
-# ... other resource blocks ...
+resource "iterative_task" "example" {
+  cloud   = "aws" # or any of: gcp, az, k8s
+  machine = "m"   # medium, or any of: l, xl, m+k80, xl+v100, ...
+
+  storage {
+    workdir = "."
+    output  = "results"
+  }
+  script = <<-END
+    #!/bin/bash
+    mkdir results
+    echo "Hello World!" > results/greeting.txt
+  END
+}
 ```
 
-**Note:** to use your local build, specify `source = "github.com/iterative/iterative"` (`source = "iterative/iterative"` will download the latest stable release instead).
+See the [Reference](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task) for the full list of options for `main.tf`.
 
 ### Initialize the provider
 
-Run this command after every `make install` to use the new build:
+Run this once to pull the latest TPI plugin:
 
 ```console
 terraform init --upgrade
```

Comment:
Does --upgrade really needed? Can we simplify the flow by getting rid of this?

Reply:
It is needed for development (after every build), not for users. We should probably remove `--upgrade` from the tutorial. 👍🏼

Reply:
If/when we release a new option (adding to the API) the user would either need `--upgrade` (or `version` in `main.tf`). Maybe not worth complicating things over.

I don't have a strong opinion.

```suggestion
terraform init
```